web/src/playground_links_popover.ts

```diff
@@ -8,12 +9,23 @@ import * as popover_menus from "./popover_menus";
 import * as realm_playground from "./realm_playground";
 import * as ui_util from "./ui_util";
 
-let playground_links_popover_instance;
+type RealmPlayground = {
+    id: number;
+    name: string;
+    pygments_language: string;
+    url_template: string;
+    playground_url?: string;
+};
```

Comment:
Don't duplicate this type with the one in `realm_playground`; import it if you want to reuse it.

Reply:
Sure .Thanks!!